### PR TITLE
[TW-1636] UI tech debt

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Screenshot updates

### CODE REVIEW

File: src/pages/docs/sending-requests/grpc/grpc-request-interface.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ Create a new gRPC request by selecting **New** in the sidebar. Select **gRPC** f
...
 
-<img src="https://assets.postman.com/postman-docs/v10/grpc-request-sections-v10-2.jpg" alt="Request pane" />
+<img src="https://assets.postman.com/postman-docs/v10/grpc-request-sections-v10-22.jpg" alt="Request pane" />
 
 * **Protocol menu** - You can change the protocol for a new request. Select **New** in the sidebar and then select a request protocol, or select **+** to open a new tab. Select the protocol dropdown menu to the left of the request's name, and then select a different request protocol.
 
     > You can't change the request protocol after you select **Save**.
 
-    <img src="https://assets.postman.com/postman-docs/v10/select-protocol-v10-2.gif" alt="Select protocol" />
+    <img src="https://assets.postman.com/postman-docs/v10/select-protocol-v10-21-11.gif" alt="Select protocol" />
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/grpc/grpc-request-interface.md:47]: In the footer, we're showing the track selector that's internal to Postman. Should we update the GIF with the footer cropped out of it?

Also, it looks like we're using the same GIF at `/docs/sending-requests/requests/#selecting-protocols`.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! I added one suggestion about the GIF.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by mark-anthony-postman — APPROVED** [review]
Other than Matthew's comment, LGTM! Thanks again for all your hard work on updating screenshots 🌟

**Comment by nerminamiller-postman** [comment]
> Other than Matthew's comment, LGTM! Thanks again for all your hard work on updating screenshots 🌟

I’ll get back to this, but for now I’ll keep plugging along.

File: src/pages/docs/sending-requests/mqtt-client/first-mqtt-request.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ To connect to an MQTT service, you need a broker URL. This example uses the publ
 
 A message in the response area indicates that you're connected to the broker.
 
-  <img src="https://assets.postman.com/postman-docs/v10/mqtt/mqtt-connected-v10-1.jpg" alt="Connect to a broker" width="500px"/>
+  <img src="https://assets.postman.com/postman-docs/v10/mqtt-connected-v10-22.jpg" alt="Connect to a broker" width="500px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/mqtt-client/first-mqtt-request.md:38]: I think you could remove the width from this and all the screenshots on this page, so they will be expandable

File: src/pages/docs/sending-requests/mqtt-client/mqtt-client-overview.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Postman provides you with a client interface to debug your MQTT-based APIs. If y
 
 You can create a new MQTT request, connect to a broker, subscribe to topics, publish messages, and view the messages received from the broker. You can save your MQTT request into a collection to reuse later, document it and share it with your teammates, and publish it to the community on [Postman's public API network](/docs/getting-started/exploring-public-api-network/).
 
-  <img src="https://assets.postman.com/postman-docs/v10/mqtt/mqtt-hero-v10-1.jpg" alt="MQTT interface" width="500px"/>
+  <img src="https://assets.postman.com/postman-docs/v10/mqtt-hero-v10-22.jpg" alt="MQTT interface" width="769px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/mqtt-client/mqtt-client-overview.md:20]:
```suggestion
  <img src="https://assets.postman.com/postman-docs/v10/mqtt-hero-v10-22.jpg" alt="MQTT interface" />
```

File: src/pages/docs/sending-requests/websocket/create-a-websocket-request.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ You can create a WebSocket request from the sidebar in Postman.
 
 1. Enter the WebSocket server URL. A WebSocket URL begins with `ws://` or `wss://`.
 
-    <img alt="WebSocket URL" src="https://assets.postman.com/postman-docs/v10/websocket-server-url-v10-19.jpg" width="600px"/>
+    <img alt="WebSocket URL" src="https://assets.postman.com/postman-docs/v10/websocket-server-url-v10-22.jpg" width="600px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/websocket/create-a-websocket-request.md:36]:
```suggestion
    <img alt="WebSocket URL" src="https://assets.postman.com/postman-docs/v10/websocket-server-url-v10-22.jpg" />
```

**Review by mark-anthony-postman — APPROVED** [review]
Looking good! Added some notes in  widths

File: src/pages/docs/sending-requests/grpc/using-grpc-examples.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ This walkthrough creates and executes a unary request, then saves the response a
 
 1. Select **Save as Example**. The saved example opens in a new tab and you can see the saved example under the request in the sidebar.
 
-    ![gRPC Save example button](https://assets.postman.com/postman-docs/v10/grpc-save-example-4saveExampleButton-2-v10.jpg)
+    ![gRPC Save example button](https://assets.postman.com/postman-docs/v10/grpc-save-example-v10-22.jpg)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/sending-requests/grpc/using-grpc-examples.md:52]: This is really minor, and might not be an issue. But I wanted to bring it up so you're aware: The request's name in the screenshot is `grpc.posthman-echo.com`, which is the server URL with a minor typo, but the value in the server URL field is correct. In the screenshot in step 3, the request's name and server URL match.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/grpc/using-grpc-examples.md (hunk 5 above)
Comment (nerminamiller-postman) [reply src/pages/docs/sending-requests/grpc/using-grpc-examples.md:52]: Thank you! Will fix!

Comment (nerminamiller-postman) [inline src/pages/docs/sending-requests/grpc/using-grpc-examples.md:52]:
```suggestion
    ![gRPC Save example button](https://assets.postman.com/postman-docs/v10/grpc-save-example-v10-22-1.jpg)
```

File: src/pages/docs/sending-requests/grpc/grpc-request-interface.md (hunk 1 above)
Comment (nerminamiller-postman) [reply src/pages/docs/sending-requests/grpc/grpc-request-interface.md]: Done!

**Review by jkonrath-postman — APPROVED** [review]

**Pull request merged**